tools/grep_cryptography.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Grep Netplan's codebase for cryptography related terms, in order to update
+# its "Cryptography" documentation in doc/security.md
+# BEWARE of --color=always when using "grep -v" to filter out stuff, as the
+# output will include control characters for the colors.
+
+GREP="grep \
```

Comment:
To improve the maintainability and readability of the script, consider breaking down the long GREP command into multiple lines or functions. This can help in understanding and modifying the script as needed. [medium]